include/mxnet/ndarray.h

```diff
@@ -875,8 +860,10 @@ class NDArray {
   size_t byte_offset_ = 0;
   /*! \brief type of data */
   int dtype_ = -1;
+  /*! \brief storage type of data */
+  NDArrayStorageType storage_type_ = kUndefinedStorage;
```

Comment:
Is the `storage_type` in Chunk removed/no longer used?????

Reply:
Still used. This is for accessing storage_type for none NDArray